packages/blockchain-wallet-v4-frontend/src/data/components/utils/selectors.ts

```diff
@@ -22,6 +23,14 @@ export const getCoinsWithBalanceOrMethod = (state: RootState) => {
   const erc20sR = selectors.core.data.eth.getErc20AccountTokenBalances(state)
   const recentSwapTxs = selectors.custodial.getRecentSwapTxs(state).getOrElse([] as SwapOrderType[])
   const custodials = selectors.core.data.coins.getCustodialCoins()
+  // TODO: Move this code to another place to clean this file
+  const userData = selectors.modules.profile.getUserData(state).getOrElse({} as UserDataType)
+  const userCountryCode = userData?.address?.country || 'default'
+  const countryCurrencies = {
+    AR: ['ARS'],
+    default: ['USD', 'EUR', 'GBP']
+  }
```

Comment:
i dont like the idea of hardcoding this list in yet another spot. please find a way to use `WalletFiatEnum` and `userCountryCode` to build the `fiatCurrencies`

Reply:
@schnogz Done!